main/src/library/Fixpoint/Ast/VarSym.flix

```diff
@@ -13,23 +13,27 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 namespace Fixpoint/Ast {
-    pub enum VarSym with Eq, Order {
+    pub enum VarSym {
```

Comment:
In fact, you can just derive Eq, Order, and ToString?

Reply:
I'm entirely sure how derive works, but would it be the same case for `RamSym`?

Reply:
Or would that automatically compare `RamSym` on all values of the typle?